filament/src/materials/ssao/sao.mat

```diff
@@ -153,8 +153,16 @@ fragment {
         return fract(m.z * fract(dot(w, m.xy)));
     }
 
-    highp vec3 computeViewSpacePositionFromDepth(highp vec2 uv, highp float linearDepth) {
-        return vec3((0.5 - uv) * materialParams.positionParams.xy * linearDepth, linearDepth);
+    highp vec2 getApiAgnosticFragCoords() {
```

Comment:
Maybe just getFragCoord()? (at least drop the s at the end)

Reply:
+1 for  `getFragCoord`

Reply:
We need a comment that explains that it's returning in the gl convention (0,0) at bottom-left.